x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -54,6 +58,9 @@ patterns. It also includes the authority to grant the privileges necessary to
 manage follower indices and auto-follow patterns. This privilege is necessary
 only on clusters that contain follower indices.
 
+`manage_enrollment`::
```

Comment:
Same as above: 

Privilege to create enrollment tokens that can be used to join new nodes to a cluster....